### DISCUSSION TITLE
Beta 2 of 3.11.0 is now out

### DISCUSSION THREAD

**Discussion by ann0see** [jamulussoftware/jamulus#3358] [Announcements]
We have tagged the second beta for Jamulus 3.11.0, [Jamulus 3.11.0beta2](https://github.com/jamulussoftware/jamulus/releases/tag/r3_11_0beta2) (version reported is just "beta", not "beta1"...). This release changes the macOS bundle identifier and will be signed. Therefore - like the final release-  it will require macOS users to recreate their local configuration or move their old configuration as documented here: https://jamulus.io/wiki/Software-Manual#backing-up-jamulus the new identifier is app.jamulussoftware.Jamulus

Please download the installer for your platform from the [release note](https://github.com/jamulussoftware/jamulus/releases/tag/r3_11_0beta2) and start using it for your normal usage of Jamulus - **Keep a back-up of your 3.10.0 install** for now.

If you find any problems, please raise a bug report making it clear that you're using the **3.11.0beta2** release.

For people who have to build from source, please note the dependencies have changed, the one major change being the minimum version of Qt is now 5.12.2.  (Builds have been with Qt up to 6.7.2.)

The updated website will be release formally once translations are reviewed, complete and the final release is announced.

**Comment by rdica** [comment]
Testing client on Ubuntu Studio 22.04, no issues so far.
Server built on Ubuntu 24.04, no issues experienced during session.

**Comment by dcorson-ticino-com** [comment]
Testing the Win10 client yesterday I saw myself on 2 channels.  After a reboot this happened again.
I am not sure that it is a problem with the client, however, as we all had sound problems with the server which seemed to be having connection problems.  After changing to another server all was well.
????

**Reply by ann0see** [reply]
Yes. You can see yourself twice if there are network issues. Can you reproduce this now again?

**Reply by dcorson-ticino-com** [reply]
It seems to be OK today, although there is more jitter than usual.
